familyconnect_registration/testsettings.py

```diff
@@ -12,3 +12,7 @@
 CELERY_ALWAYS_EAGER = True
 BROKER_BACKEND = 'memory'
 CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
+
+PASSWORD_HASHERS = (
+    'django.contrib.auth.hashers.MD5PasswordHasher',
+)
```

Comment:
Yup, ideally we would want to switch the password hasher off for tests, since for most tests we need to create at least one user, and hashing the password is intentionally computationally expensive. But since we can't switch it off, we just change it to the cheapest option.